packages/mds-policy/api.ts

```diff
@@ -243,6 +247,18 @@ function api(app: express.Express): express.Express {
     })
     .unknown(true) // TODO
 
+  app.post(pathsFor('/policies/:policy_id'), (req, res) => {
```

Comment:
Stick this behind an admin endpoint for the time being, don't want this publicly exposed.

Reply:
👍